streamwebs_frontend/streamwebs/views.py

```diff
@@ -24,11 +24,203 @@
     TransectZone, Canopy_Cover, CameraPoint, PhotoPoint,
     PhotoPointImage, Soil_Survey, Resource, School)
 
+from djqscsv import render_to_csv_response
+import csv
 import json
 import copy
 import datetime
 
 
+def export_wq(request, site_slug):
```

Comment:
Removing the request param results in this error:
![remove_request](https://cloud.githubusercontent.com/assets/6801364/24305930/e50b21e2-107b-11e7-96d0-95651cfe99a6.png)